File: python/program/Pelanggan.py
class Pelanggan:
    def __init__(self, id_pelanggan="", nama="", kontak=""):
        self.id_pelanggan = id_pelanggan
        self.nama = nama
        self.kontak = kontak

    # Getter
    def get_id_pelanggan(self):
        return self.id_pelanggan

    def get_nama(self):
        return self.nama

    def get_kontak(self):
        return self.kontak

    # Setter
    def set_id_pelanggan(self, id_pelanggan):
        self.id_pelanggan = id_pelanggan

    def set_nama(self, nama):
        self.nama = nama

    def set_kontak(self, kontak):
        self.kontak = kontak

File: java/program/Pelanggan.java
public class Pelanggan {
    private String idPelanggan;
    private String nama;
    private String kontak;

    public Pelanggan(String idPelanggan, String nama, String kontak) {
        this.idPelanggan = idPelanggan;
        this.nama = nama;
        this.kontak = kontak;
    }

    // Getter
    public String getIdPelanggan() {
        return idPelanggan;
    }

    public String getNama() {
        return nama;
    }

    public String getKontak() {
        return kontak;
    }

    // Setter
    public void setIdPelanggan(String idPelanggan) {
        this.idPelanggan = idPelanggan;
    }

    public void setNama(String nama) {
        this.nama = nama;
    }

    public void setKontak(String kontak) {
        this.kontak = kontak;
    }
}
